### PR TITLE
feat(transpile): Auto-run & enable Heroku deploy

### PR DESCRIPTION
**Pull request by jameswomack**
* Use $PORT
* Auto-run after 1s
* Don't watch on `npm start`
* Dedicated local start task
* Conform to semver

This might address issue #3 in the source repo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "esnextbin",
-  "version": "0.1.0",
+  "version": "1.0.0-alpha.0",
```
Comment (voronianski) [inline package.json]: @jameswomack what's the point of changing this? I feel completely fine with `0` version for now.

**Comment by voronianski** [comment]
@jameswomack I think this should be an optional. For example some checkbox to enable/disable it.

File: src/containers/Main.jsx [hunk 2]
```diff
@@ -157,6 +158,14 @@ class Main extends React.Component {
         }
         const editorsData = this._updateEditorsData({code, transpiledCode, error: ''});
         this.setState({ editorsData });
+
+        if (this.timeout) {
+          clearTimeout(this.timeout);
```
Comment (voronianski) [inline src/containers/Main.jsx]: @jameswomack please check `.editorconfig` - https://github.com/voronianski/esnextbin/blob/master/.editorconfig#L6

**Comment by voronianski** [comment]
@jameswomack I've merged with several updates.

**Comment by jameswomack** [comment]
Thanks for the merge @voronianski. This a cool product you've built. 

Regarding the version change: It was a zero cost change that makes the version number valid against the semantic versioning spec NPM uses*. Therefore humans using NPM can make valid assumptions about your version number according to their understanding of semver. 

*Version numbers below 1 aren't valid according to that spec. Prerelease versions follow the format I committed according to that spec.

**Comment by jameswomack** [comment]
Thanks for adding the auto run section in the readme

**Pull request merged**